ecosystem/indexer-grpc/indexer-grpc-utils/src/file_store_operator_v2/file_store_operator.rs

```diff
@@ -0,0 +1,84 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::file_store_operator_v2::common::BatchMetadata;
+use anyhow::Result;
+use aptos_protos::transaction::v1::Transaction;
+use prost::Message;
+use tokio::sync::mpsc::Sender;
+
+pub struct FileStoreOperatorV2 {
+    max_size_per_file: usize,
+    num_txns_per_folder: u64,
+
+    buffer: Vec<Transaction>,
+    buffer_size: usize,
```

Comment:
nit: `buffer_size_in_bytes`? `size` may be confusing in different contexts.

Reply:
Done